plugins/oaiMetadataFormats/marcxml/templates/record.tpl

```diff
@@ -25,9 +25,9 @@
 			<subfield code="a">{$journal->getData('printIssn')|escape}</subfield>
 		</datafield>
 	{/if}
-	{if $article->getStoredPubId('doi')}
+	{if $publication->getStoredPubId('doi')}
```

Comment:
We now use `$publication->getDoi()`.
`getStoredPubId('doi')` is old -- from the time when DOI was a plugin.